ENIGMAsystem/SHELL/Widget_Systems/widgets_mandatory.h

```diff
@@ -23,7 +23,21 @@
 
 #include <string>
 
-#define DEBUG_MESSAGE(msg, severity) ::enigma_user::show_debug_message((std::string) (msg) + " | " __FILE__ ":" + std::to_string(__LINE__), (severity))
+#ifdef ABORT_ALL_ERRORS
+  #ifdef HIDE_DEBUG_MESSAGES
+    #define DEBUG_MESSAGE(msg, severity) if (severity >= 2) { exit(1); } \
```

Comment:
The debug message won't be printed for severe errors. It'll just silently exit. Also, you should use a constant instead of 2.